### ISSUE TITLE
Gmail import from K9 failed

### ISSUE THREAD

**Issue by fnxweb** [thunderbird/thunderbird-android#8312]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

8.0b2

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Pixel 6 Pro

### Steps to reproduce

Installed Thunderbird beta
Select import from K9 Mail
Follow Google OAuth dialogues for Gmail account

### Expected behavior

Gmail account available

### Actual behavior

Gmail account reported login failure. 
Managed to trigger option to sign in again (from error?) but this created a second Gmail account entry in Thunderbird.

Deleted the duplicate and selected the original and managed to sign in finally.

**Comment by mutaunt** [comment]
I have experienced this issue importing K9 settings too. It affected both a gmail and a yahoo account, so may be related to the OAuth2 authentication requiring google or yahoo logins.

After logging in to google/yahoo and allowing Thunderbird access the accounts both appeared in my list of Thunderbird accounts but when selected only showed an outbox.

When I attempted to look at server settings for each it triggered issue #8321, I was prompted to create new email account.

Creating both the gmail and yahoo accounts from scratch worked to create a second but functional account for each.
